src/HtmlUtils.js

```diff
@@ -208,17 +210,25 @@ module.exports = {
             finally {
                 delete sanitizeHtmlParams.textFilter;
             }
-            return <span className="markdown-body" dangerouslySetInnerHTML={{ __html: safeBody }} />;
         } else {
-            safeBody = content.body;
+            safeBody = _.escape(content.body);
```

Comment:
The only major difference here should be the handing of plantext bodies. Our goal is to embed images in them, which means they must be rendered as HTML, so they need to be escaped (which wasn't needed earlier.) Which is why the `_.escape` is only in the plaintext code path. 

By the time we're at L225, if we had an HTML body, it's been sanitized and highlighted, and if we had a plaintext body, it's been escaped and highlighted. L225 is problematic though, since it assumes it's getting a string. I'll fix this in another commit - but it'll most likely mean that we'll have to use HtmlHighlighter for plaintext as well, but then it's pretty much the same since we've escaped that plaintext.